overview/default.nix

```
@@ -146,34 +148,79 @@ let
         '';
     };
 
-    projects = rec {
+    projects = {
       one = name: project: ''
-        <section><details><summary>${heading 2 name}</summary>
+        ${heading 1 name}
         <https://nlnet.nl/project/${name}>
 
         ${render.packages.many (pick.packages project)}
         ${render.options.many (pick.options project)}
         ${render.examples.many (pick.examples project)}
-        </details></section>
       '';
-      many = projects: concatLines (mapAttrsToList one projects);
+      # Many projects are renderes as links to their individual project sites
+      many =
+        projects:
+        concatLines (
+          mapAttrsToList (name: _: ''
+            <a href="${name}">${name}</a>
```

We can make the page link take us directly to the project page (see https://github.com/ngi-nix/ngipkgs/pull/570#issuecomment-2713083802):

```suggestion
            <a href="${name}/index.html">${name}</a>
```

---

Thanks, I'll respond to your review here. Opening the pages as `file://` in the browser doesn't work and I think it's reasonable to require contributors to use some kind of minimal webserver (e.g. I use `darkhttpd`). Opening as `file://` also brings the disadvantage of not allowing AJAX and even is inconsistently implemented in different browsers.
On a different note, I'd rather not have `index.html` fragments in URLS. These are implementation details and shouldn't be visible to the user (and they make the URL longer).

---

Alright, that seems reasonable to me. Then what do you think about adding another package that runs a HTTP server on the overview? Something like `overview-preview` that users can run to see how it would look in production.

We could alternatively rename `overview` to `overview-static` and have `overview` be the package that starts the server.

---

Mh how about we just add `darkhttpd` to `devShells.default` and document how to build the overview and run the server?

---

Sure, would that be better for reloading changes faster?